js/validation/validationSignUp.js

```diff
@@ -0,0 +1,157 @@
+    surname.classList.add('form__input_error')
+    errorMessageSurname.innerHTML = 'Please enter a valid surname.'
+    errorMessageSurname.classList.add('form__text_error')
+    return false
+  } else {
+    surname.classList.remove('form__input_error')
+    errorMessageSurname.innerHTML = ''
+  }
+
+  if (!email.value) {
+    email.classList.add('form__input_error')
+    errorMessageEmail.innerHTML = 'Please enter your email.'
+    errorMessageEmail.classList.add('form__text_error')
+    return false
+  } else {
+    email.classList.remove('form__input_error')
+    errorMessageEmail.innerHTML = ''
+  }
+
+  if (!emailIsValid(email.value)) {
+    email.classList.add('form__input_error')
+    errorMessageEmail.innerHTML = 'Please enter a valid email.'
+    errorMessageEmail.classList.add('form__text_error')
+
+    return false
+  } else {
+    email.classList.remove('form__input_error')
+    errorMessageEmail.innerHTML = ''
+  }
+
+  if (!password.value) {
+    password.classList.add('form__input_error')
+    errorMessagePassword.innerHTML = 'Please enter your password.'
+    errorMessagePassword.classList.add('form__text_error')
+    return false
+  } else {
+    password.classList.remove('form__input_error')
+    errorMessagePassword.innerHTML = ''
+  }
+
+  if (!passwordIsValid(password.value)) {
+    password.classList.add('form__input_error')
+    errorMessagePassword.innerHTML = 'Please enter a valid password.'
+    errorMessagePassword.classList.add('form__text_error')
+
+    return false
+  } else {
+    password.classList.remove('form__input_error')
+    errorMessagePassword.innerHTML = ''
+  }
+
+  if (!confirmPass.value) {
+    confirmPass.classList.add('form__input_error')
+    errorMessageConfirmPass.innerHTML = 'Please enter your password.'
+    errorMessageConfirmPass.classList.add('form__text_error')
+    return false
+  } else {
+    confirmPass.classList.remove('form__input_error')
+    errorMessageConfirmPass.innerHTML = ''
+  }
+
+  if (!passwordIsValid(confirmPass.value)) {
+    confirmPass.classList.add('form__input_error')
+    errorMessageConfirmPass.innerHTML = 'Please enter a valid password.'
+    errorMessageConfirmPass.classList.add('form__text_error')
+
+    return false
+  } else {
+    confirmPass.classList.remove('form__input_error')
+    errorMessageConfirmPass.innerHTML = ''
+  }
+
+  if (!passEqual(password.value)) {
+    password.classList.add('form__input_error')
+    errorMessagePassword.innerHTML = 'Passwords not equal'
+    errorMessagePassword.classList.add('form__text_error')
+    confirmPass.classList.add('form__input_error')
+    errorMessageConfirmPass.innerHTML = 'Passwords not equal'
+    errorMessageConfirmPass.classList.add('form__text_error')
+    return false
+  } else {
+    password.classList.remove('form__input_error')
+    errorMessagePassword.innerHTML = ''
+    confirmPass.classList.remove('form__input_error')
+    errorMessageConfirmPass.innerHTML = ''
+  }
+
+  return true
```

Comment:
The same situation from the upper cases. You have to remove duplicates in code, and probably create one place for all fields in your project.